cv32e40s/tests/programs/custom/hpmcounter_hazard_test/hpmcounter_hazard_test.c

```diff
@@ -60,7 +60,7 @@ int main(int argc, char *argv[])
 
   volatile unsigned int minstret;
 
-  __asm__ volatile(".option rvc");
+  __asm__ volatile(".option rvc");  // TODO:ERROR:silabs-robin  Why? Should be in Yaml cfg.
```

Comment:
Remove, this should be handled in bsp code

Reply:
Fixed. The 3 tests in question still pass.